src/lib/components/Timeline/timeline.tsx

```diff
@@ -252,59 +248,34 @@ export const Timeline: React.FC<TimelineProps> = (props: TimelineProps): JSX.Ele
         };
     }, [setSliderActive]);
 
-    const handleDateChange = React.useCallback(
-        (date: MaterialUiPickersDate) => {
-            if (date) {
-                setCurrentDate(date.startOf("day"));
-            }
-        },
-        [setCurrentDate]
-    );
+    const handleDateChange = (e: DateEvent) => {
+        if (e?.target?.value) setCurrentDate(dayjs(e.target.value));
+    };
 
     React.useEffect(() => {
         if (props.onDateChange && currentDate) {
             props.onDateChange(currentDate);
         }
     }, [currentDate]);
 
-    const toggleDatePickerDialogVisibility = () => {
-        setDatePickerOpen(!datePickerOpen);
-    };
-
     return (
         <div className="FlowDiagramExplorer__Timeline">
             {currentDate && (
                 <div className="FlowDiagramExplorer__Timeline__CurrentSelectionLabel">
-                    <Tooltip title="Open date picker dialog" placement="top">
-                        <Button
-                            className="FlowDiagramExplorer__Timeline__Toggle"
-                            variant="ghost_icon"
-                            onClick={toggleDatePickerDialogVisibility}
-                        >
-                            <Icon name="calendar" title="Current date" size={16} />
-                        </Button>
-                    </Tooltip>
-                    <MuiPickersUtilsProvider utils={DayjsUtils}>
-                        <DatePicker
-                            variant="inline"
-                            value={currentDate}
-                            onChange={handleDateChange}
-                            format="MMMM DD, YYYY"
-                            open={datePickerOpen}
-                            maxDate={
-                                sortedTimeFrames.length > 0
-                                    ? sortedTimeFrames[sortedTimeFrames.length - 1].endDate
-                                    : undefined
-                            }
-                            onOpen={() => setDatePickerOpen(true)}
-                            onClose={() => setDatePickerOpen(false)}
-                            minDate={sortedTimeFrames.length > 0 ? sortedTimeFrames[0].startDate : undefined}
-                            className="FlowDiagramExplorer__Timeline__DatePicker"
-                            InputProps={{
-                                disableUnderline: true,
-                            }}
-                        />
-                    </MuiPickersUtilsProvider>
+                    <TextField
+                        id="timeline_date"
+                        type="date"
```

Comment:
Does this work well? Did you test different browsers?

https://caniuse.com/input-datetime

Reply:
yeah good point 😅 Seems to be working well on the major browsers. 

Setting the input type to `week` and `month` isn't supported in all browsers, but we're not using this. Min/Max isn't supported in some niche browsers, but that doesn't break anything it's just a worse user experience _(which you're probably used to if you're using those browsers)_

So i guess EDS could fix the missing browser support themselves, but I'm guessing it's not worth the time for them.